en/scale-a-tidb-cluster.md

```diff
@@ -1,60 +1,74 @@
 ---
-title: Scale TiDB in Kubernetes
-summary: Learn how to horizontally and vertically scale up and down a TiDB cluster in Kubernetes.
+title: Manually Scale TiDB in Kubernetes
+summary: Learn how to manually scale a TiDB cluster in Kubernetes.
 aliases: ['/docs/tidb-in-kubernetes/dev/scale-a-tidb-cluster/']
 ---
 
-# Scale TiDB in Kubernetes
+# Manually Scale TiDB in Kubernetes
 
 This document introduces how to horizontally and vertically scale a TiDB cluster in Kubernetes.
 
 ## Horizontal scaling
 
-Horizontally scaling TiDB means that you scale TiDB out or in by adding or remove nodes in your pool of resources. When you scale a TiDB cluster, PD, TiKV, and TiDB are scaled out or in sequentially according to the values of their replicas. Scaling out operations add nodes based on the node ID in ascending order, while scaling in operations remove nodes based on the node ID in descending order.
+Horizontally scaling TiDB means that you scale TiDB out or in by adding or remove Pods in your pool of resources. When you scale a TiDB cluster, PD, TiKV, and TiDB are scaled out or in sequentially according to the values of their replicas.
 
-Currently, the TiDB cluster supports management by TidbCluster Custom Resource (CR).
+- To scale out a TiDB cluster, **increase** the value of `replicas` of a certain component. The scaling out operations add Pods based on the Pod ID in ascending order, until the number of Pods equals the value of `replicas`.
 
-### Scale PD, TiDB, and TiKV
+- To scale in a TiDB cluster, **decrease** the value of `replicas` of a certain component. The scaling in operations remove Pods based on the Pod ID in descending order, until the number of Pods equals the value of `replicas`.
 
-Modify `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` in the `TidbCluster` object of the cluster to a desired value using kubectl. You can modify the values in the local file or using online command.
+### Horizontally scale PD, TiKV, and TiDB
 
-- You can also online modify the `TidbCluster` definition in the Kubernetes cluster by running the following command:
+To scale PD, TiKV, or TiDB horizontally, use kubectl to modify `spec.pd.replicas`, `spec.tikv.replicas`, and `spec.tidb.replicas` in the `TidbCluster` object of the cluster to a desired value. You can modify the values in the local file or using online command.
+
+1. Modify the `replicas` value of a component as needed. For example, running the following command can dynamically configure the `replicas` value of PD to 3:
```

Comment:
```suggestion
1. Modify the `replicas` value of a component as needed. For example, configure the `replicas` value of PD to 3:
```

Reply:
I'm not sure if this is "dynamically".

Reply:
> I'm not sure if this is "dynamically".

@ran-huang According to [the existing Chinese version](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/scale-a-tidb-cluster#%E6%89%A9%E7%BC%A9%E5%AE%B9-pdtidbtikv), it is.